tgstation.dme

```diff
@@ -5635,7 +5635,6 @@
 #include "modular_skyrat\master_files\code\modules\mob\living\emote_popup.dm"
 #include "modular_skyrat\master_files\code\modules\mob\living\examine_tgui.dm"
 #include "modular_skyrat\master_files\code\modules\mob\living\living_defines.dm"
-#include "modular_skyrat\master_files\code\modules\mob\living\carbon\death.dm"
```

Comment:
why, just comment block out this file we arnt getting this as a merge conflict later